Makefile

```diff
@@ -13,6 +13,7 @@ help:
 	@echo "make test        Test cookbook for all platforms."
 	@echo "make py          Build the Cookbook for Python only."
 	@echo "make r           Build the Cookbook for R only."
+	@echo "make java        Build the Cookbook for Java only."
```

Comment:
`make java` builds the HTML output, but how do we test that the examples still work?

Reply:
FYI, I think that the original idea was to have the code in `.java` files that we could run to verify the examples and then use `:literalinclude:` to embed the function that should go into the recipe.

I don't know if `jshell` opened for use additional possibilities, but we need an answer on how the testing part should be done.